### PR TITLE
[tests-only][full-ci]add test for updating content of file

### PR DESCRIPTION
**Pull request by anon-pradip**
## Description
This PR tests the updating of content of a file inside the personal space as to reflect new content edits.

## Related Issue

- Part of https://github.com/owncloud/web/issues/10259

## Motivation and Context

- e2e test for updating the content of file has not been implemented

## How Has This Been Tested?

- Locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests
- [ ] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

### CODE REVIEW

File: tests/e2e/cucumber/features/shares/share.feature [hunk 1]
```diff
@@ -81,6 +81,10 @@ Feature: share
       | shareToBrian.md     | mdFile     | readme    |
       | shareToBrian.drawio | drawioFile |           |
       | sharedFile.txt      | txtFile    | some text |
+    And "Alice" edits the following resources
```
Comment (SagarGi) [inline tests/e2e/cucumber/features/shares/share.feature:84]: is there any other feature file where we could cover up the tests for this edit seems odd here though.

**Review by SagarGi — COMMENTED** [review]
other LGTM :+1:

File: tests/e2e/cucumber/features/shares/share.feature (hunk 1 above)
Comment (anon-pradip) [reply tests/e2e/cucumber/features/shares/share.feature:84]: For the time being, I see no other suitable feature files to put the test for editing content of file. If later added, we can re-use this test.

**Review by SagarGi — APPROVED** [review]
LGTM :+1:

**Review by saw-jan — APPROVED** [review]

**Comment by saw-jan** [comment]
Port to the stable-8 branch please (PR should be in the stable branch first)

**Pull request merged**